flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/data/util/DataFormatConverters.java

```diff
@@ -900,6 +902,36 @@ Timestamp toExternalImpl(RowData row, int column) {
         }
     }
 
+    /** Converter for timestamp which doesn't consider the time zone. */
+    public static final class TimestampLtzConverter
```

Comment:
Please also update the non-legacy `org.apache.flink.table.data.conversion.DataStructureConverters`